shelley/chain-and-ledger/executable-spec/test/Test/Shelley/Spec/Ledger/Rules/TestPool.hs

```diff
@@ -132,17 +124,15 @@ poolRetireInEpoch env ssts =
                 )
                   ==> ( eval (certWit ∈ dom stp)
                           && eval (certWit ∈ dom stp')
-                          && Maybe.maybe False ((== cepoch) . epochFromSlotNo) (stp' !? certWit)
```

Comment:
@uroboros - if you could make sure it is okay to remove this line, I would appreciate it. We no longer care to know the slot when a pool is registered, so I think this is not needed.

Reply:
I am amazed at how many files this change touched. Everything looks good to me.